src/utils/viewer/viewer.cpp

```diff
@@ -61,6 +61,10 @@
 #include "esp/sensor/FisheyeSensor.h"
 #include "esp/sim/Simulator.h"
 
+// #ifdef ESP_BUILD_WITH_AUDIO // todo sangarg
```

Comment:
todo  : Check where this flags should be enabled. Adding in viewer/CMakeLists.txt was not working